content/en/tracing/profiler/connect_traces_and_profiles.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Connect Traces and Profiles
+kind: Documentation
+further_reading:
+    - link: 'tracing'
+      tag: 'Documentation'
+      text: 'APM Distributed Tracing'
+    - link: 'tracing/profiler/getting_started'
+      tag: 'Documentation'
+      text: 'Enable Continuous Profiler for Your Application.'
+    - link: 'tracing/profiler/intro_to_profiling'
+      tag: 'Documentation'
+      text: 'Introduction to Profiling'
+---
+
+If your application is showing performance problems in production, it's helpful to connect the distributed trace information from APM with the full profile of the code stack. Application processes that have both APM distributed tracing and continuous profiler enabled are automatically linked, so you can move directly from span information to profiling data on the Code Hotspots tab, and find specific lines of code related to performance issues.
+
+{{< img src="tracing/profiling/code_hotspots_tab.gif" alt="Code Hotspots tab shows profiling information for a APM trace span">}}
+
+## Prerequisites
+
+{{< programming-lang-wrapper langs="java,python" >}}
+{{< programming-lang lang="java" >}}
+For manually instrumented code, Continuous Profiler requires Scope activation of spans:
+```java
+final Span span = tracer.buildSpan("ServicehandlerSpan").start();
+try (final Scope scope = tracer.activateSpan(span)) { // mandatory for Datadog continuous profiler to link with span
+    // worker thread impl
+  } finally {
+    // Step 3: Finish Span when work is complete
+    span.finish();
+  }
+
+```
+{{< /programming-lang >}}
+{{< programming-lang lang="python" >}}
+
+- Requires tracing library version [0.44.0+].
+
+{{< /programming-lang >}}
+{{< /programming-lang-wrapper >}}
+
+## Link from a span to profiling data
+
+From the view of each trace, the Code Hotspots tab highlights profiling data scoped on the selected span
+
+The breakdown view on the left side is a list of types of time spent executing that span. Depending on the runtime and language, this list of types varies:
+
+- **Method durations** shows the overall time taken by each method from your code.
+- **CPU** shows the time taken executing CPU tasks.
+- **Synchronization** shows the time taken waiting for a lock of a synchronized object.
+- **Garbage collection** shows the time taken waiting for the garbage collector to execute.
+- **VM operations** (Java only) shows the time taken waiting for VM operations that are not related to garbage collection (for example, heap dumps).
+- **File I/O** shows the time taken waiting for a disk read/write operation to execute.
+- **Socket I/O** shows the time taken waiting for a network read/write operation to execute.
+- **Object wait** shows the time ...
```

Comment:
@hdafgard what should we put here?

Reply:
It's related to the Synchronization data, so something like 
`**Object wait** shows the time taken waiting for a notify call on an object.` or something along those lines.

https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/lang/Object.html#wait()